.circleci/config.base.yml

```diff
@@ -80,6 +80,8 @@ jobs:
               - equal: [ *windows_node12, << parameters.os >> ]
           steps:
             - checkout
+            - run: nvm install 12.22.7
+            - run: nvm alias default 12.22.7
```

Comment:
I am assuming these are necessary because XCode 13 does not have node 12 bundled?

Reply:
That's right, and I pinned for windows as well since they're not explicitly assigned like they are w/ the linux versions.